src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -771,7 +772,7 @@ void AppendOrderBy()
             // Handle Multiple Results sets to include from
             if (_includeFromCteIds?.Count > 1 && _curFromCteIndex >= 0 && _curFromCteIndex < _includeFromCteIds.Count - 1)
             {
-                StringBuilder.Append($"),{Environment.NewLine}");
+                StringBuilder.AppendLine($"),");
```

Comment:
Nit: This can be a normal string now, the $ isn't needed.

Reply:
Thanks!